src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -617,6 +668,8 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             variant (`str`, *optional*):
                 Load weights from a specified variant filename such as `"fp16"` or `"ema"`. This is ignored when
                 loading `from_flax`.
+            dduf (`str`, *optional*):
+                Load weights from the specified dduf archive or folder.
```

Comment:
```suggestion
            dduf_file (`str`, *optional*):
                Load weights from the specified dduf file.
```

Same here, I would also rename it `dduf_file` for consistency within `diffusers` and with `transformers`'s `.from_pretrained(model_id, gguf_file=filename)`

Reply:
Sounds good to me. I didn't put the same name before as I extracted the archive and it ended up being a folder.